### ISSUE TITLE
H:\steve\Desktop\New folder\Climacell_2020-12-24_143004.txt

### ISSUE THREAD

**Issue by SteveSBE** [bpaauwe/udi-climacell-poly#7]
Hi - 

I am trying to use Climacell Weather on my Polisy mostly for the forecast information. (I use WeatherLink to get to my local Davis weather station current weather information.) Climacell will not load data into the ISY. I've restarted the node server several times. I also restarted my Universal Devices ISY.  But I still don't get forecast data from Climacell. A configuration screen shot and my recent log file is attached.  FWIW...In the log I notice a message indicating that "...string indices must be integeres". 

Just so you know I'm new with Polisy but thought I'd ask this question...thanks for your patience.

Thanks, 

Steve

![image](https://user-images.githubusercontent.com/31939275/103105947-f2a36d00-45f6-11eb-81ec-74ca5b369197.png)

**Comment by bpaauwe** [comment]
I'm seeing the same problem.  But when I access my account on Climacell, it says my plan has expired, but doesn't provide any info on what to do to renew it.  

It may be API key related.

**Comment by bpaauwe** [comment]
I think I figured out what's going on.

The node server was written to support version 3 of the API.  They are now on version 4 of the API and it's quite different. 

Anyone that signed up while version 3 was the current version has an account that's tied to that version.  Mine was and because it expired, it was no longer valid.  

Anyone that signs up now gets an account that works only with version 4 of the API, not with the version 3 that the node server is using.

I've started updating the node server to use version 4 of the API, but it's going to be a bit of work.  I'm also not sure if switching to version 4 will then break all existing users, requiring everyone to get a new version 4 account.  As accounts expire, they'll probably have to anyway, but it's kind of a mess.
